.vscode/settings.json

```diff
@@ -23,4 +23,13 @@
         "editor.trimAutoWhitespace": false,
         "files.trimTrailingWhitespace": false
     }
+    "terminal.integrated.shell.windows": "C:\\Program Files\\QMK_MSYS\\usr\\bin\\bash.exe",
```

Comment:
Could the changes to this file be reverted?

Reply:
That file shouldnt have uploaded. Adding to gitignore